### DISCUSSION TITLE
And-Or (like ?: in C)

### DISCUSSION THREAD

**Discussion by MrZ626** [26F-Studio/Techmino#242] [Score Board (death count)]
MrZ: 1

**Comment by flaribbit** [comment]
https://github.com/26F-Studio/Techmino/blob/3c02c5957eee16e5e4c36bb92479975bbd08abe9/Zframework/require.lua#L10-L12

[a comment by an automated account is not shown]

**Reply by Trebor-Huang** [reply]
It might not be *the* ternary operator in some languages; but in most conventional languages it is, so whatever :P
